src/modules/update_project/app/update_project_controller.py

```diff
@@ -0,0 +1,53 @@
+from src.shared.helpers.errors.controller_errors import MissingParameters
+from src.shared.helpers.errors.domain_errors import EntityError
+from src.shared.helpers.errors.usecase_errors import NoItemsFound
+from src.shared.helpers.external_interfaces.external_interface import IRequest, IResponse
+from src.shared.helpers.external_interfaces.http_codes import OK, NotFound, BadRequest, InternalServerError
+from .update_project_viewmodel import UpdateProjectViewmodel
+from .update_project_usecase import UpdateProjectUsecase
+
+class UpdateProjectController:
+    def __init__(self, usecase: UpdateProjectUsecase) -> None:
+        self.UpdateProjectUsecase = usecase
+
+
+    def __call__(self, request: IRequest) -> IResponse:
+        try:
+            code = request.data.get('code')
+            new_name = request.data.get('new_name')
+            new_description = request.data.get('new_description')
+            new_po_RA = request.data.get('new_po_RA')
+            new_scrum_RA = request.data.get('new_scrum_RA')
+            new_photos = request.data.get('new_photos')
+
+            if code is None:
+                raise MissingParameters('code')
+            
+            update_project = self.UpdateProjectUsecase(
+                code=code,
+                new_name=new_name if new_name is not None else None,
```

Comment:
Esses if verificando se é none não precisava fazer, concorda se for None ele já vai ser None, não tem que fazer o IF